src/Sulu/Bundle/TagBundle/Resources/config/lists/tags.xml

```diff
@@ -60,6 +61,11 @@
 
                 <joins ref="changer"/>
             </field>
+
+            <filter type="selection">
```

Comment:
you did not add this filter to other lists. is there a reason for adding it here specifically?

Reply:
Because it seems that this is the only file where this property exists :slightly_smiling_face: